pegasus/test/test_helper.rb

```diff
@@ -2,10 +2,10 @@
 require_relative '../../shared/test/common_test_helper'
 require_relative '../../shared/test/capture_queries'
 
-# Set up JUnit output for Circle
+# Set up JUnit output for CI
 reporters = [Minitest::Reporters::SpecReporter.new]
-if ENV['CIRCLECI']
-  reporters << Minitest::Reporters::JUnitReporter.new("#{ENV.fetch('CIRCLE_TEST_REPORTS', nil)}/pegasus")
+if CI.ci_job_ui_tests?
```

Comment:
No, VERY confusingly CIRCLECI **actually** means "running on CI and running UI tests", hence the very intentional rename here for clarity. I took a long time unwinding this, and weirdly I think one of the major improvements of this PR is dropping CIRCLECI in favor of a variable name that makes sense.

Reply:
Unit tests will not run if you treat ENV['CIRCLECI'] to just mean "running on CI", these are explicitly UI-test-only codepaths.

Reply:
Lol, brutal. I initially was concerned that we had changed the logic here because `ci_job_ui_tests?` is checking something different than what we used to check here, but it makes sense to me that the change was intentional and that the new logic is more accurate than the old logic :+1: